src/main/java/org/apache/maven/plugins/artifact/buildinfo/BuildinfoMojo.java

```diff
@@ -350,8 +351,8 @@ private void compareWithReference( Map<Artifact, String> artifacts, File referen
                 p.println( "version=" + project.getVersion() );
                 p.println( "ok=" + ok );
                 p.println( "ko=" + ko );
-                p.println( "okFiles=\"" + StringUtils.join( okFilenames.iterator(), " " ) + '"' ); 
-                p.println( "koFiles=\"" + StringUtils.join( koFilenames.iterator(), " " ) + '"' ); 
+                p.println( "okFiles=\"" + StringUtils.join( okFilenames, " " ) + '"' ); 
```

Comment:
Notice how Commons-lang StringUtils.join supports Iterable, from Java 1.5; and maven-shared-utils StringUtils.join only supports Iterator from Java 1.2. It's a minor detail, but it does show the relative amounts of maintenance and attention these two products receive.